<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>admin_tarjeta_crud</class>
 <widget class="QWidget" name="admin_tarjeta_crud">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>304</width>
    <height>147</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>303</width>
    <height>147</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>281</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
background-color: #F4F7F4;
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="btnMasPlatillo">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>70</y>
      <width>39</width>
      <height>35</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>35</width>
      <height>35</height>
     </size>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">
border: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../Recursos.qrc">
      <normaloff>:/iconos/resources/editarPlato.png</normaloff>:/iconos/resources/editarPlato.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>35</width>
      <height>35</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="nombrePlatillo">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>30</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#585858;
</string>
    </property>
    <property name="text">
     <string>Sopa de Codito</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="precioPlatillo_2">
    <property name="geometry">
     <rect>
      <x>114</x>
      <y>70</y>
      <width>16</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#858585;
</string>
    </property>
    <property name="text">
     <string>$</string>
    </property>
   </widget>
   <widget class="QLabel" name="precioPlatillo">
    <property name="geometry">
     <rect>
      <x>133</x>
      <y>70</y>
      <width>20</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#858585;
</string>
    </property>
    <property name="text">
     <string>35</string>
    </property>
   </widget>
   <widget class="QLabel" name="imgPlatillo">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>10</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>90</width>
      <height>90</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>95</width>
      <height>95</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
background-image: url(:/iconos/resources/menuPrueba.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../Recursos.qrc"/>
 </resources>
 <connections/>
</ui>
